src/main/resources/project/server/startHostController.pl

```diff
@@ -0,0 +1,630 @@
+        }
+    }
+}
+
+sub check_server_boot_errors_via_cli {
+    my %args = @_;
+    my $jboss = $args{jboss} || croak "'jboss' is required param";
+    my $host_name = $args{host_name} || croak "'host_name' is required param";
+    my $server_name = $args{server_name} || croak "'server_name' is required param";
+
+    $jboss->log_info("Checking boot errors of server '$server_name' on host '$host_name' via CLI");
+
+    my $cli_command = "/host=$host_name/server=$server_name/core-service=management/:read-boot-errors";
+
+    my %result = $jboss->run_command($cli_command);
+
+    if ($result{code}) {
+        $jboss->log_warning("Cannot read boot errors of server '$server_name' on host '$host_name' via CLI");
+        $jboss->add_summary("Cannot read boot errors of server '$server_name' on host '$host_name' via CLI");
+        $jboss->add_status_warning();
+        return;
+    }
+    else {
+        my $json = $jboss->decode_answer($result{stdout});
+        if ($json && exists $json->{result}) {
+            my $boot_errors_result = $json->{result};
+            if (!@$boot_errors_result) {
+                $jboss->log_info("No boot errors of server '$server_name' on host '$host_name'");
+                $jboss->add_summary("No boot errors of server '$server_name' on host '$host_name'");
+                return;
+            }
+        }
+
+        $jboss->log_warning("Detected boot errors of server '$server_name' on host '$host_name': " . $result{stdout});
+        $jboss->add_summary("Detected boot errors of server '$server_name' on host '$host_name', see log for details");
+        $jboss->add_status_warning();
+        return;
+    }
+}
+
+sub show_logs_via_cli_for_server {
+    my %args = @_;
+    my $jboss = $args{jboss} || croak "'jboss' is required param";
+    my $host_name = $args{host_name} || croak "'host_name' is required param";
+    my $server_name = $args{server_name} || croak "'server_name' is required param";
+
+    my $assumption_sting = sprintf(
+        "assumption is that log file is %s, tailing %u lines",
+        EXPECTED_SERVER_LOG_FILE_NAME,
+        NUMBER_OF_LINES_TO_TAIL_FROM_LOG
+    );
+
+    $jboss->log_info("Showing logs via CLI for server '$server_name' on host '$host_name' ($assumption_sting)");
+
+    my $cli_command = sprintf(
+        "/host=%s/server=%s/subsystem=logging/log-file=%s/:read-log-file(lines=%u,skip=0)",
+        $host_name,
+        $server_name,
+        EXPECTED_SERVER_LOG_FILE_NAME,
+        NUMBER_OF_LINES_TO_TAIL_FROM_LOG
+    );
+
+    my %result = $jboss->run_command($cli_command);
+    if ($result{code}) {
+        $jboss->log_warning("Cannot read logs via CLI for server '$server_name' on host '$host_name', please refer to JBoss logs on file system for more details");
+    }
+    else {
+        $jboss->log_info("JBoss logs for server '$server_name' on host '$host_name' ($assumption_sting): " . $result{stdout});
+    }
+}
+
+sub escape_additional_options_for_windows {
```

Comment:
Please, move that to JBoss driver.

Reply:
I am considering this function as a very specific for only two procedures: StartStandaloneServer and StartHostController (and only for Windows startup) that's why I suggest to leaver this function here, at least while we do not have separate libs for procedures (or group of procedures) within jboss_driver properysheet